src/rpm/package.rs

```diff
@@ -843,8 +843,13 @@ impl PackageMetadata {
             Err(Error::TagNotFound(_)) => Ok(None),
             Err(e) => return Err(e),
         };
+        let links = self
+            .header
+            .get_entry_data_as_string_array(IndexTag::RPMTAG_FILELINKTOS);
```

Comment:
Presuming an RPM has no files with symlinks whatsoever, is `RPMTAG_FILELINKTOS` guaranteed to exist?  If yes, this is fine, if no, then it should be treated like `RPMTAG_FILECAPS` a few lines above so that we don't erroneously return a bad result.

Reply:
Don't know based on the documentation: https://rpm-software-management.github.io/rpm/manual/tags.html. However, I've just added a new symlink-free test-case and the tag is present. Hope it helps.

Reply:
It looks like it adds the tag if there are files (regardless if any are symlinks), and always does if there are, and doesn't otherwise.  But in the event that there are no files then we should return no file entries, so it's fine.  

TL;DR this is OK, so long as there aren't RPMs floating around with different behavior.